src/types/PerformanceTable.cpp

```diff
@@ -395,4 +427,23 @@ void PerformanceTable::display() {
       std::cout << std::endl;
     }
   }
-}
\ No newline at end of file
+}
+
+bool PerformanceTable::operator==(const PerformanceTable &pt) const {
+
+  if (this->pt_.size() != pt.getPerformanceTable().size() &&
+      this->pt_[0].size() != pt.getPerformanceTable().size()) {
```

Comment:
shouldn't the check here be `pt_.size != pt.size || pt_[0].size != pt[0].size` ?

Reply:
you have to call getPerformanceTable() for pt.

Reply:
The comment was about the logic (&& -> ||) and I believe you forgot a [0] in the second part

Reply:
yup thanks for that I got tired probably ahah.